src/ontology/cl-edit.owl

```diff
@@ -33025,6 +33026,14 @@ AnnotationAssertion(rdfs:label obo:CL_4047034 "smooth muscle cell of stomach"@en
 SubClassOf(obo:CL_4047034 obo:CL_0000192)
 SubClassOf(obo:CL_4047034 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0008856))
 
+# Class: obo:CL_4047036 (stomach smooth muscle circular layer cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11146428") Annotation(oboInOwl:hasDbXref "PMID:32424507") obo:IAO_0000115 obo:CL_4047036 "A smooth muscle cell located in the middle layer of the muscularis externa of the stomach wall. This cell is arranged concentrically with the stomach's longitudinal axis, forming a continuous sheet of contractile tissue. It is fusiform in shape, containing actin and myosin filaments that enable contraction without striations. This cell contributes to the mechanical digestion and movement of food within the stomach through coordinated contractions, and plays a role in forming the pyloric sphincter in the pyloric region.")
+AnnotationAssertion(terms:contributor obo:CL_4047036 <https://orcid.org/0009-0005-7919-4905>)
+AnnotationAssertion(terms:date obo:CL_4047036 "2024-12-09T10:40:58Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4047036 "stomach smooth muscle circular layer cell"@en)
+SubClassOf(obo:CL_4047036 obo:CL_4047034)
```

Comment:
Make EquivalentClass Def (smooth muscle and part_of some 'stomach smooth muscle circular layer')